hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/BackendSessionPool.java

```diff
@@ -37,12 +39,14 @@
     private final String name;
     private final ThreadLocal<BackendSession> threadLocalSession;
     private final AtomicInteger sessionCount;
+    private final Map<Long, BackendSession> sessions;
```

Comment:
seems can remove `ThreadLocal<BackendSession> threadLocalSession` if added `Map<Long, BackendSession> sessions`, if it does, add a TODO